coops-ui/src/views/AnnualReport.vue

```diff
@@ -36,113 +36,140 @@
       <v-container id="annual-report-container" class="view-container">
         <v-row>
           <v-col cols="12" lg="9">
-            <article id="annual-report-article" :class="this.agmDate ? 'agm-date-selected' : 'no-agm-date-selected'">
-              <header>
-                <h1 id="AR-header">File {{ ARFilingYear }} Annual Report
-                  <span class="font-italic" v-if="reportState">- {{ reportState }}</span>
-                </h1>
-                <p>Please verify or change your Office Addresses and Directors.</p>
-              </header>
-
-              <div v-if="isAnnualReportEditable">
-                <!-- Annual General Meeting Date ( COOP ) -->
-                <section v-if="entityFilter(EntityTypes.COOP)">
-                  <header>
-                    <h2 id="AR-step-1-header">1. Annual General Meeting Date</h2>
-                    <p>Select your Annual General Meeting (AGM) date</p>
-                  </header>
-                  <AGMDate
-                    :initialAgmDate="initialAgmDate"
-                    :allowCOA="allowChange('coa')"
-                    :allowCOD="allowChange('cod')"
-                    @agmDate="agmDate=$event"
-                    @noAGM="noAGM=$event"
-                    @valid="agmDateValid=$event"
-                  />
-                </section>
-
-                <!-- Annual Report Date ( BCORP ) -->
-                <section v-if="entityFilter(EntityTypes.BCORP)">
-                  <header>
-                    <h2 id="AR-step-1-header-BC">1. Dates</h2>
-                    <p>Your Annual Report Date is the anniversary of the date your corporation was started.<br>
-                      The information displayed on this form reflects the state of your corporation on this
-                      date each year.</p>
-                  </header>
-                  <ARDate />
-                </section>
-
-                <!-- Registered Office Addresses -->
-                <section>
-                  <header>
-                    <h2 id="AR-step-2-header">2. Registered Office Addresses
-                      <span class="agm-date">(as of {{ ARFilingYear }} Annual General Meeting)</span>
-                    </h2>
-                    <p>Verify or change your Registered Office Addresses.</p>
-                  </header>
-                  <OfficeAddresses
-                    :changeButtonDisabled="!allowChange('coa')"
-                    :addresses.sync="addresses"
-                    :registeredAddress.sync="registeredAddress"
-                    :recordsAddress.sync="recordsAddress"
-                    @modified="officeModifiedEventHandler($event)"
-                    @valid="addressFormValid = $event"
-                  />
-                </section>
+            <article>
+            <!-- Annual Report Date ( COOP ) -->
+              <article
```

Comment:
"Article" is an HTML5 semantic element, and I think it doesn't make sense to nest one article inside another. Perhaps the outer one should be a section? See https://www.w3schools.com/html/html5_semantic_elements.asp
Or, perhaps the outer article doesn't need to exist?

Reply:
Great suggestion 👍  I updated and wrapped the articles in a <section>

Reply:
Great!